packages/core/e2e/default-search-plugin.e2e-spec.ts

```diff
@@ -1592,25 +1592,34 @@ describe('Default search plugin', () => {
                 await awaitRunningJobs(adminClient);
             });
 
+            it('fallbacks to default language', async () => {
+                const { search } = await searchInLanguage(LanguageCode.af)
+                // No records for AF language, but we expect > 0
+                // because of fallback to default language (EN)
+                expect(search.totalItems).toBeGreaterThan(0);
+            })
+
             it('indexes product-level languages', async () => {
                 const {search: search1} = await searchInLanguage(LanguageCode.de);
 
-                expect(search1.items[0].productName).toBe('laptop name de');
-                expect(search1.items[0].slug).toBe('laptop-slug-de');
-                expect(search1.items[0].description).toBe('laptop description de');
+                expect(search1.items.map(i => i.productName)).toContain('laptop name de');
```

Comment:
What's the reason for no longer using the zero'th index here? Does the new query alter the order of results?

Reply:
The reason is that prior to this change there was only one search record, because there was no fallback on default language. So checking with 0 index may return a row in default language, rather than yours. I don't think this can be counted as altering of result's order, rather than additional search results.

Reply:
About 1.7: I partially agree with you, it introduces slightly a new behavior, but current behavior is buggy in term of searching on multi-language environment. Imagine you work with any i18n solution: when you request a translation by key, but you don't have a translation for that, i18n will just return you a default translation, this is some kind of fallback. Also I can say that other graphql queries, like `products()`, `collections()` successfully fallbacks to default language if translation is missing.

Reply:
```This change will also make the DefaultSearchPlugin behave differently to the ElasticsearchPlugin, so I think we will need to update that one too to also fall back to the default translations. I'm not sure if you have experience with ES but let me know if you'd like to take that on too.```
I'm new to ES, maybe there are somebody other people who could work around this :)